learning-outcomes.md

```diff
@@ -0,0 +1,58 @@
+# Week 1 Learning Outcomes
+
+## Teamwork & Toolkit
+
+### Pair programming
+
+- [ ] **We can use pair programming techniques to work together**
+- [ ] We can work on one computer as a pair
+- [ ] We can use four different paired programming techniques
+- [ ] We can distribute commits evenly among team members
+
+---
+
+### Version control
+
+- [ ] **We can use Git and GitHub as version control for our code**
+- [ ] We can initialise a Git repository so that we can track changes to our code over time
+- [ ] We can add / stage files so that we can include these updates in our next commit
+- [ ] We can write descriptive commit messages so that our commit history tells a story
+- [ ] We can amend commit messages so that we can correct typos/mistakes before we push to GitHub
+- [ ] We can co-author commit messages so that our commit history reflects everyone’s contributions
+- [ ] We can merge master to our feature branch so that we can resolve any merge conflicts locally
+- [ ] We can resolve merge conflicts to maintain a working codebase
+- [ ] We can push and pull branches from a remote to work on different tasks
+- [ ] We can create a pull request on GitHub to request feedback from our teammates
+- [ ] We can review other people’s pull requests to provide feedback on their code
+- [ ] We can delete branches that have been merged to keep our repo tidy
+
+---
+
+### Accessibility
+
+- [ ] **We can make a website that is accessible by people other than ourselves**
+- [ ] We can structure a site using semantic HTML to aid accessibility
+- [ ] We can add appropriate attributes to HTML elements to make them screen reader friendly
+- [ ] We can add ARIA labels (where necessary) to make our semantic HTML more accessible
+- [ ] We can run a Lighthouse accessibility audit to check our website’s accessibility
+- [ ] We can use a screen reader to understand how people with visual impairments maye experience our website
+- [ ] We can implement a suitable contrast ratio on our website so that it meets the needs of people who are colour blind
+- [ ] We can use various tools (Lighthouse, Chromevox, WCAG Contract Checker Extension, Devtools) to check that our website meets accessibility criteria
```

Comment:
Chromevox is a weird one. I don't think anyone actually uses it as assistive tech—it seems to only be used by devs doing testing. It's ok for getting the general idea of what a screenreader is, but they're all so different and have such specific requirements that it seems like a waste to teach something that no one really uses.

Would it be unfair to ask Mac users to use Voiceover and have Linux users stick with ChromeVox? (Since JAWS is v expensive)

Reply:
That's interesting - I'll omit to specify the screenreader that should be used then